integration/jnifuse/native/src/main/native/libjnifuse/operation.h

```diff
@@ -105,7 +105,7 @@ class MkdirOperation : public Operation {
 class RenameOperation : public Operation {
  public:
   RenameOperation(JniFuseFileSystem *fs);
-  int call(const char *oldPath, const char *newPath);
+  int call(const char *oldPath, const char *newPath, unsigned int flags);
```

Comment:
Can you help recreate the libjnifuse.so and libjnifuse3.so files after removing the double comma?

Reply:
The md5 value for new libjnifuse.so and new libjnifuse3.so have not changed. This error should be added after the files are generated.